docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -1,185 +1,228 @@
+Follow these steps to prepare the system:
 
-This guide will only get ONE node (previously minion) working.  Multiple nodes require a functional [networking configuration](/docs/concepts/cluster-administration/networking/) done outside of Kubernetes.  Although the additional Kubernetes configuration requirements should be obvious.
+1. Make sure that operating system software is up to date. On Fedora Atomic, type:
 
-The Kubernetes package provides a few services: kube-apiserver, kube-scheduler, kube-controller-manager, kubelet, kube-proxy.  These services are managed by systemd and the configuration resides in a central location: `/etc/kubernetes`.  We will break the services up between the hosts.  The first host, fed-master, will be the Kubernetes master.  This host will run the kube-apiserver, kube-controller-manager, and kube-scheduler.  In addition, the master will also run _etcd_ (not needed if _etcd_ runs on a different host but this guide assumes that _etcd_ and Kubernetes master run on the same host).  The remaining host, fed-node will be the node and run kubelet, proxy and docker.
+    <pre><tt># <b>atomic host upgrade</b>
+   # <b>systemctl reboot</b>
+    </tt></pre>
 
-**System Information:**
+   On a Fedora Server or Fedora Workstation system, type:
 
-Hosts:
+    <pre><tt># <b>dnf update -y</b>
+   # <b>systemctl reboot</b>
+    </tt></pre>
 
-```conf
-fed-master = 192.168.121.9
-fed-node = 192.168.121.65
-```
+    Once the system comes up, log in again and run the next steps.
 
-**Prepare the hosts:**
+2. Disable swap. Comment out (#) any swap areas from /etc/fstab. For example, a commented out swap line in /etc/fstab might appear as follows:
 
-* Install Kubernetes on all hosts - fed-{master,node}.  This will also pull in docker. Also install etcd on fed-master.  This guide has been tested with Kubernetes-0.18 and beyond.
-* Running on AWS EC2 with RHEL 7.2, you need to enable "extras" repository for yum by editing `/etc/yum.repos.d/redhat-rhui.repo` and changing the `enable=0` to `enable=1` for extras.
+    <pre><tt>
+    <b>#</b> /dev/mapper/fedora--atomic-swap swap  swap  defaults  0 0
+    </tt></pre>
 
-```shell
-dnf -y install kubernetes
-```
+3. Disable selinux. Change the SELINUX type in /etc/sysconfig/selinux to permissive, so it appears as follows:
 
-* Install etcd
+        SELINUX=permissive
 
-```shell
-dnf -y install etcd
-```
+4. Immediately disable swap and SELinux:
 
-* Add master and node to `/etc/hosts` on all machines (not needed if hostnames already in DNS). Make sure that communication works between fed-master and fed-node by using a utility such as ping.
+    <pre><tt>
+    # <b>swapoff -a</b>
+    # <b>setenforce 0</b>
+    </tt></pre>
 
-```shell
-echo "192.168.121.9    fed-master
-192.168.121.65    fed-node" >> /etc/hosts
-```
+5. Disable firewalld. On Fedora Server or Workstation only (firewalld is no on Fedora Atomic by default), type the following.
```

Comment:
nit, replace "no on" with "not enabled on"

Reply:
Good catch. It should be "is not enabled on Fedora".